source/about/history.rst

```diff
@@ -0,0 +1,83 @@
+=====================
+The History of Sponge
+=====================
+
+This page is split into two main parts:
+ * the history of the project itself
+ * the history of our beloved mascot Spongie
+
+The History of the Project
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sponge was founded as a better alternative to the APIs that were available as of
+September 2014. The Sponge :doc:`staff` consists of many people from
+different Communities, eg. Spout/Spoutcraft, Forge, Cauldron and a few others.
+
+When the Bukkit, Cauldron and MCPC+ DMCA and the development halt of Bukkit
+happened the Minecraft community was shocked. Several developers from the
+above mentioned communities gathered in #nextstep on Esper.NET and discussed
+the future of Minecraft modding. It was decided that there definitely is a
+demand for a new API as the Bukkit repository (Cauldron and MCPC+ too)
```

Comment:
"certainly" would be better than "definitely", and "since" would be nicer than "as"

> *It didn't take long to come to the conclusion that there certainly was a demand for a new API, since Bukkit's repository - and Cauldron's too - has been taken down."*

Since Cauldron and MCPC+ are the same thing, mentioning them separately doesn't really make sense either.